process_blocking.go

```diff
@@ -385,7 +385,7 @@ func (p *blockingProcess) Wait(ctx context.Context) (int, error) {
 		case p.ops <- waiter:
 			continue
 		case <-ctx.Done():
-			return -1, errors.New("wait operation canceled")
+			return -1, errors.Wrap(ctx.Err(), "operation canceled")
```

Comment:
Just return `ctx.Err()`

Reply:
Same as above. I wrapped it with a better message.